src/main/java/core/basesyntax/figure/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.figure;
+
+public class Rectangle extends AbstractFigure implements Figure {
```

Comment:
The Rectangle class should not extend AbstractFigure as per the checklist. Instead, use interfaces to define behaviors.